components/surveyWidget/widgetBox/styles.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright 2020 Inrupt Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to use,
+ * copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the
+ * Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+ * PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+import { createStyles } from "@solid/lit-prism-patterns";
+
+const BOTTOM_POSITION_CLOSE_BUTTON = "88%";
+const LEFT_POSITION_CLOSE_BUTTON = "81%";
+const SHADOW = "0 6px 23px 0 rgba(149, 149, 149, 0.2)";
+const DARK_TEAL_TEXT_COLOR = "073D6C";
+
+export default function styles(theme) {
+  return createStyles(theme, ["button"], {
+    widgetBox: {
```

Comment:
nitpick: I notice all the other keys are hyphenated. We should keep these consistent. I would personally prefer using camel case to avoid all the quoting but it doesn't really matter either way.

Reply:
The reason for this is how we use the classes to override MUI vs bem, but I switched it to hyphenated for consistency.